### PR TITLE
915138_ocr2 Upgrade GitHub demo samples to the latest .net version [PDF Library]

### PR DESCRIPTION
**Pull request by sameerkhan001**
Hi All,

Task id: https://dev.azure.com/EssentialStudio/Document%20Processing%20Libraries/_workitems/edit/915138/

Task name: Upgrade GitHub demo samples to the latest .net version [PDF Library]

Regards,
Sameerkhan N

### CODE REVIEW

File: ASP.NET MVC/Perform_OCR_MVC/packages.config [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="2.0.1" targetFramework="net462" />
+  <package id="Microsoft.jQuery.Unobtrusive.Validation" version="4.0.0" targetFramework="net462" />
+  <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net462" />
+  <package id="Modernizr" version="2.8.3" targetFramework="net462" />
+  <package id="Newtonsoft.Json" version="13.0.3" targetFramework="net462" />
+  <package id="Syncfusion.Compression.Base" version="27.1.53" targetFramework="net462" />
+  <package id="Syncfusion.Licensing" version="27.1.53" targetFramework="net462" />
+  <package id="Syncfusion.Pdf.AspNet.Mvc5" version="27.1.53" targetFramework="net462" />
+  <package id="Syncfusion.Pdf.OCR.AspNet.Mvc5" version="27.1.53" targetFramework="net462" />
+  <package id="Syncfusion.Pdf.OCR.Wpf" version="27.1.53" targetFramework="net462" />
```
Comment (chinnumuniyappan) [inline ASP.NET MVC/Perform_OCR_MVC/packages.config:20]: May I know the reason for using this WPF package?

File: ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj [hunk 2]
```diff
@@ -49,17 +49,14 @@
...
     </Reference>
-    <Reference Include="Syncfusion.Compression.Base, Version=26.2450.8.0, Culture=neutral, PublicKeyToken=3d67ed1f87d44c89, processorArchitecture=MSIL">
-      <HintPath>..\packages\Syncfusion.Pdf.AspNet.Mvc5.26.2.8\lib\net45\Syncfusion.Compression.Base.dll</HintPath>
+    <Reference Include="Syncfusion.Licensing, Version=27.1462.53.0, Culture=neutral, PublicKeyToken=632609b4d040f6b4, processorArchitecture=MSIL">
+      <HintPath>..\packages\Syncfusion.Licensing.27.1.53\lib\net462\Syncfusion.Licensing.dll</HintPath>
     </Reference>
-    <Reference Include="Syncfusion.Licensing, Version=26.2450.8.0, Culture=neutral, PublicKeyToken=632609b4d040f6b4, processorArchitecture=MSIL">
-      <HintPath>..\packages\Syncfusion.Pdf.AspNet.Mvc5.26.2.8\lib\net45\Syncfusion.Licensing.dll</HintPath>
+    <Reference Include="Syncfusion.OCRProcessor.Base, Version=27.1462.53.0, Culture=neutral, PublicKeyToken=3d67ed1f87d44c89, processorArchitecture=MSIL">
+      <HintPath>..\packages\Syncfusion.Pdf.OCR.Wpf.27.1.53\lib\net462\Syncfusion.OCRProcessor.Base.dll</HintPath>
```
Comment (chinnumuniyappan) [inline ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj:56]: The project is MVC, But have you restored the WPF package?

File: ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj [hunk 3]
```diff
@@ -249,14 +246,14 @@
       </FlavorProperties>
     </VisualStudio>
   </ProjectExtensions>
+  <Import Project="..\packages\Syncfusion.Pdf.OCR.AspNet.Mvc5.27.1.53\lib\net462\Syncfusion.Pdf.OCR.AspNet.Mvc5.targets" Condition="Exists('..\packages\Syncfusion.Pdf.OCR.AspNet.Mvc5.27.1.53\lib\net462\Syncfusion.Pdf.OCR.AspNet.Mvc5.targets')" />
+  <Import Project="..\packages\Syncfusion.Pdf.OCR.Wpf.27.1.53\build\net462\Syncfusion.Pdf.OCR.Wpf.targets" Condition="Exists('..\packages\Syncfusion.Pdf.OCR.Wpf.27.1.53\build\net462\Syncfusion.Pdf.OCR.Wpf.targets')" />
```
Comment (chinnumuniyappan) [inline ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj:250]: No need to install WPF package

**Review by chinnumuniyappan — CHANGES_REQUESTED** [review]

File: ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj (hunk 2 above)
Comment (sameerkhan001) [reply ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj]: Removed

File: ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj (hunk 3 above)
Comment (sameerkhan001) [reply ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj]: removed

File: ASP.NET MVC/Perform_OCR_MVC/packages.config (hunk 1 above)
Comment (sameerkhan001) [reply ASP.NET MVC/Perform_OCR_MVC/packages.config]: removed

File: ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj [hunk 4]
```diff
@@ -249,14 +253,16 @@
...
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
-    <Error Condition="!Exists('..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.2.0.1\build\net46\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.2.0.1\build\net46\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.props'))" />
-    <Error Condition="!Exists('..\packages\Syncfusion.Pdf.OCR.AspNet.Mvc5.26.2.8\build\net45\Syncfusion.Pdf.OCR.AspNet.Mvc5.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Syncfusion.Pdf.OCR.AspNet.Mvc5.26.2.8\build\net45\Syncfusion.Pdf.OCR.AspNet.Mvc5.targets'))" />
+    <Error Condition="!Exists('..\packages\Syncfusion.Pdf.OCR.Wpf.27.1.53\build\net462\Syncfusion.Pdf.OCR.Wpf.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Syncfusion.Pdf.OCR.Wpf.27.1.53\build\net462\Syncfusion.Pdf.OCR.Wpf.targets'))" />
```
Comment (chinnumuniyappan) [inline ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj:262]: Why still WPF targets is added here?

**Review by chinnumuniyappan — CHANGES_REQUESTED** [review]

File: ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj (hunk 4 above)
Comment (sameerkhan001) [reply ASP.NET MVC/Perform_OCR_MVC/Perform_OCR_MVC.csproj]: Removed

**Review by chinnumuniyappan — APPROVED** [review]

**Pull request merged**